12-Capstone/deploy-aws-eb.md

```diff
@@ -1,111 +1,42 @@
+## Learning Goals
 
-RDS allows you to quickly provision a database of several different flavors including PostgreSQL and MySQL. You provide the size of database you want plus the username and password for logging in, and AWS sets it up and gives you a specific domain name for accessing it.
+After following along with this lesson you should be able to:
 
-While this is relatively straight forward, Elastic Beanstalk again makes our lives easier by integrating this setup process into the application configuration web page. Furthermore, when we have an RDS database connected to our application then EB will automatically configure a set of environment variables with the information needed for our code to connect to that database.
+1. Follow a guide to deploy a Ruby on Rails web application to EB
+1. Troubleshoot issues with deployment involving the database.
+1. Manage environment variables in an EB instance.  
 
-#### Heroku
-Now that we've taken a pretty significant tour through the different systems that Elastic Beanstalk uses to manage our deployments, lets take a look at how this differs with Heroku.
+## Tutorial Overview
 
-While it may seem like there's a lot more involved in an Elastic Beanstalk application, in truth both of these systems are very similar! Heroku's system involves every one of the above components and uses them in almost exactly the same ways.
+In this tutorial we will clone a rails application, set up an Elastic Beanstalk instance on AWS and deploy the application to that instance. 
 
-The main difference between the two is that Heroku hides many of the underlying systems from developers which makes learning to use it and deploy your first application significantly faster. The other side to that coin however, is that you have less control over how each component of your deployment stack is configured and Heroku must explicitly support the kind of application you're attempting to deploy.
+## Step 1:  Get the application code
 
-## Live Demonstration
-### Get the application code
 Clone the repository from https://github.com/adagold/aws-eb-rails (there is no need to fork this).
 
 **Make sure to run `bundle install`**
 
-### Setup the EB application
+## Step 2: Setup the EB application
+
 1. Log into the [AWS Management Console](https://console.aws.amazon.com/).
-1. Go to the [Elastic Beanstalk Management Console](https://us-west-2.console.aws.amazon.com/elasticbeanstalk/home?region=us-west-2#/getting_started) (we're using the Oregon datacenter aka us-west-2).
-1. Create a new EB application:
+2. Go to the [Elastic Beanstalk Management Console](https://us-west-2.console.aws.amazon.com/elasticbeanstalk/home?region=us-west-2#/getting_started) (we're using the Oregon datacenter aka us-west-2).
+3. Create a new EB application:
```

Comment:
If you use `1.` for all the numbers, the markdown compiler will figure out the numbering automatically. Makes it much easier to insert into a list.

```md
1. first
1. second
1. third
```

yields

1. first
1. second
1. third